fofix/core/Theme.py

```diff
@@ -1933,21 +1933,21 @@ def renderAlbumArt(self, scene):
                     if abs(d) < 1.2:
                         # Fetch item's label image
                         label = scene.itemLabels[i]
-                        # Replace with default if needed
-                        if label == "Random":
-                            if scene.img_random_label:
-                                label = scene.img_random_label.texture
-                            else:
-                                label = None
-                        if not label:
-                            if scene.img_empty_label:
-                                label = scene.img_empty_label.texture
+                        # Compatibility: replace str with label
+                        if label == "Random" and scene.img_random_label:
```

Comment:
Combining those two conditions will remove the `label` if the condition is `False`, and then will remove non-empty labels. I think you should separate your two conditions instead ;).